src/templates/upload-dropdown-menu.html

```diff
@@ -14,6 +14,20 @@
       </button>
     {% endif %}
 
+    {# INSERT IN TEXT in edit mode for videos #}
+    {% if ext matches '/(mpeg|mpg|mpe|mp4|ogv|webm|avi)$/i' %}
+      <button type='button' class='dropdown-item' data-action='insert-video-in-body' data-name='{{ upload.real_name|e('html_attr') }}' data-storage='{{ upload.storage }}' data-link='{{ upload.long_name|e('html_attr') }}' data-uploadid='{{ upload.id }}'>
+        <i class='fas fa-fw fa-video mr-1'></i>{{ 'Insert in text at cursor position'|trans }}
+      </button>
+    {% endif %}
+
+    {# INSERT IN TEXT in edit mode for audio #}
+    {% if ext matches '/(mp3|ogg|opus|wav|flac)$/i' %}
```

Comment:
it's missing aac

and you should order them alphabetically

Reply:
Sure, will add and sort. Eventually we're running into the same problem as with videos here as well only to a smaller extend... we're matching file extensions, not codecs or even better mime types.